.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -146,11 +146,12 @@ jobs:
       run: cargo install sccache mdbook mdbook-linkcheck
 
     - name: "Compiler cache"
-      # Only restore here, later we save for only the main branch
-      # and always use the same key as we start from the main branch cache
-      uses: actions/cache/restore@v3
+      uses: actions/cache@v3
```

Comment:
i don't think this worked fully, as it didn't update the cache "not saving cache" ?

```
Post job cleanup.
Cache hit occurred on the primary key Ubuntu [2](https://github.com/KDAB/cxx-qt/actions/runs/4850575112/jobs/8643604078#step:32:2)2.04 (gcc) Qt5_compiler_cache, not saving cache.
```